src/main/python/ttconv/vtt/paragraph.py

```diff
@@ -61,9 +64,14 @@ def get_end(self) -> Optional[ClockTime]:
     """Returns the paragraph end time code"""
     return self._end
 
-  def is_only_whitespace(self):
-    """Returns whether the paragraph tex contains only whitespace"""
-    return self._text.isspace()
+  def is_only_whitespace_or_empty(self):
+    """Returns whether the paragraph text contains only whitespace or is empty"""
+    return len(self._text) == 0 or self._text.isspace()
+
+  def normalize_eol(self):
+    """Remove line breaks at the beginning and end of the paragraph, and replace
+    line break sequences with a single line break"""
+    self._text = VttParagraph._EOL_SEQ_RE.sub("\n", self._text).strip("\n\r")
```

Comment:
@palemieux I think #350 is not critical as `xml:preserve` is not used (at least from observation) very often, if at all. It is also not recommended to use it because the result can not be predicted and depends on the selection of fonts. So I think it is safe to wait with this issue and prioritize other issues higher. If the review is OK for this PR it is ready to be merged from my view.